From 590b70738f434d5cd244d5d774960afc459c90d2 Mon Sep 17 00:00:00 2001
From: Troy Ronda <t.....@securekey.com>
Date: Tue, 19 Sep 2017 21:01:32 -0400
Subject: [PATCH] Update package channelconfig for vendoring

TODO: Script extraction

Copyright SecureKey Technologies Inc. All Rights Reserved.
SPDX-License-Identifier: Apache-2.0

Signed-off-by: Troy Ronda <t.....@securekey.com>
---
 common/channelconfig/keys.go | 73 ++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 73 insertions(+)
 create mode 100644 common/channelconfig/keys.go

diff --git a/common/channelconfig/keys.go b/common/channelconfig/keys.go
new file mode 100644
index 00000000..0aab0f6c
--- /dev/null
+++ b/common/channelconfig/keys.go
@@ -0,0 +1,73 @@
+/*
+Copyright SecureKey Technologies Inc, IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package channelconfig
+
+//////////////////
+// Extracted from applicationorg.go
+const (
+	// AnchorPeersKey is the key name for the AnchorPeers ConfigValue
+	AnchorPeersKey = "AnchorPeers"
+)
+
+//////////////////
+// Extracted from channel.go
+const (
+	// ConsortiumKey is the key for the cb.ConfigValue for the Consortium message
+	ConsortiumKey = "Consortium"
+
+	// HashingAlgorithmKey is the cb.ConfigItem type key name for the HashingAlgorithm message
+	HashingAlgorithmKey = "HashingAlgorithm"
+
+	// BlockDataHashingStructureKey is the cb.ConfigItem type key name for the BlockDataHashingStructure message
+	BlockDataHashingStructureKey = "BlockDataHashingStructure"
+
+	// OrdererAddressesKey is the cb.ConfigItem type key name for the OrdererAddresses message
+	OrdererAddressesKey = "OrdererAddresses"
+
+	// GroupKey is the name of the channel group
+	ChannelGroupKey = "Channel"
+)
+
+//////////////////
+// Extracted from msp_util.go
+const (
+	// ReadersPolicyKey is the key used for the read policy
+	ReadersPolicyKey = "Readers"
+
+	// WritersPolicyKey is the key used for the read policy
+	WritersPolicyKey = "Writers"
+
+	// AdminsPolicyKey is the key used for the read policy
+	AdminsPolicyKey = "Admins"
+
+	// MSPKey is the org key used for MSP configuration
+	MSPKey = "MSP"
+)
+
+//////////////////
+// Extracted from orderer.go
+const (
+	// OrdererGroupKey is the group name for the orderer config
+	OrdererGroupKey = "Orderer"
+)
+
+const (
+	// ConsensusTypeKey is the cb.ConfigItem type key name for the ConsensusType message
+	ConsensusTypeKey = "ConsensusType"
+
+	// BatchSizeKey is the cb.ConfigItem type key name for the BatchSize message
+	BatchSizeKey = "BatchSize"
+
+	// BatchTimeoutKey is the cb.ConfigItem type key name for the BatchTimeout message
+	BatchTimeoutKey = "BatchTimeout"
+
+	// ChannelRestrictionsKey is the key name for the ChannelRestrictions message
+	ChannelRestrictionsKey = "ChannelRestrictions"
+
+	// KafkaBrokersKey is the cb.ConfigItem type key name for the KafkaBrokers message
+	KafkaBrokersKey = "KafkaBrokers"
+)
-- 
2.14.1

